client-mvp-04/src/store/configureStore.js

```diff
@@ -1,15 +1,11 @@
-  
-import { createStore, combineReducers } from 'redux';
-import authReducer from '../reducers/authReducer';
+import { createStore } from 'redux';
+import allReducers from '../reducers/index';
```

Comment:
Please, remove `index` from import. If we have index.js, it's enough to write :
`import allReducers from '../reducers';`